docs/install/Knative-custom-install.md

```diff
@@ -66,9 +66,7 @@ The following Knative installation files are available:
   - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin.yaml
   - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin-in-mem.yaml
 - **Eventing Component**:
-  - https://github.com/knative/eventing/releases/download/{{< version >}}/release.yaml
```

Comment:
Just to make sure I understand, @yolocs  -- going forward, the only artifact released for eventing will be eventing.yaml, and it will come bundled with the in-memory channel provisioner? No more option to install it separately?

Reply:
Yes, that's my understanding. @grantr

Reply:
Cool, just wanted to make sure I understood what was changing :) I'll wait to merge this until the Eventing PR is merged.

Reply:
FYI. The eventing PR has been merged.

Reply:
> the only artifact released for eventing will be eventing.yaml, and it will come bundled with the in-memory channel provisioner? No more option to install it separately?

No, this is incorrect. There will still be 3 artifacts:
- `eventing-core.yaml` will be just the eventing core without InMemoryChannel, equivalent to the previous `eventing.yaml`
`in-memory-channel.yaml` will be just the InMemoryChannel, unchanged
- `eventing.yaml` will be eventing-core.yaml and in-memory-channel.yaml combined, equivalent to the previous `release.yaml`

Reply:
If I understand it correctly, only `eventing.yaml` will actually be useful, right? I don't see serving list serving-core.yaml here.

Reply:
You're right, serving-core.yaml isn't documented here so eventing-core.yaml can also be undocumented. I was just answering @samodell's question about installing separately. That option will still exist, even if undocumented.

Reply:
Thanks for clarifying.